Makefile

```diff
@@ -1,17 +1,17 @@
 # Copyright 2021 Absa Group Limited
-# 
+#
```

Comment:
The PR looks LGTM for me, but we must do any license changes in golic/config.yaml. 

The problem is, when license string differs from template, it injects new license. Here, we will have two licenses in makefile as a result. 

Could you please, in the scope of this PR, skip license change?

Reply:
@kuritka hmm, might be my editor acting and auto-removing extra spaces, and diff ignores this change. will surely amend

Reply:
@kuritka amended